### PR TITLE
add spectrogram plugin

### PR DESCRIPTION
**Pull request by akreal**
This is most likely not ready for merge yet, just wanted to let you know that the work is going and probably to get some early feedback.

### CODE REVIEW

**Comment by jczimm** [comment]
+1 for this, sounds great. :+1:

File: example/spectrogram/index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <title>wavesurfer.js | Spectrogram plugin</title>
+
+        <link href="data:image/gif;" rel="icon" type="image/x-icon" />
+
+        <!-- Bootstrap -->
+        <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
+        <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
```
Comment (thijstriemstra) [inline example/spectrogram/index.html]: These bootstrap links are old, check out the latest examples.

**Comment by thijstriemstra** [comment]
@akreal could you make sure you're only using spaces, there's a lot of odd indentation when looking at the diffs.

**Comment by akreal** [comment]
I've messed up totally this PR and created #337 instead.

**Pull request closed without merging**